examples/running_flct.py

```diff
@@ -0,0 +1,122 @@
+"""
+===================================
+Fourier Local Correlation Tracking
+===================================
+
+This example applies Fourier Local Correlation Tracking (FLCT)
+to a set of two images taken within a short interval of each other
+using `~sunkit_image.flct.flct`.
+"""
+
+import numpy as np
+
+from sunkit_image.flct.flct import flct
+import matplotlib.pyplot as plt
+
+###########################################################################
+# This examples demonstrates how to find the
+# 2D velocity flow field between two images taken within a short span of time.
+# We will create two dummy images containing a moving
+# object and assume the time difference between the images to be one second.
+
+# Creating the input arrays
+image1 = np.zeros((10, 10))
+image1[0:3, 0:3] = 1
+
+image2 = np.zeros((10, 10))
+image2[1:4, 1:4] = 1
+
+# Plot both the images
+fig = plt.figure(figsize=(10, 4))
+
+ax1 = fig.add_subplot(121)
+plt.imshow(image1)
+ax1.set_title("First Image")
+
+ax2 = fig.add_subplot(122)
+plt.imshow(image2)
+ax2.set_title("Second Image")
```

Comment:
I don't see the purpose of this part of the code.

Reply:
I was just showing both the input images together